apps/openmw/mwmp/LocalPlayer.cpp

```diff
@@ -1119,7 +1119,7 @@ void LocalPlayer::setEquipment()
             else
             {
                 // Don't try to equip an item that is already equipped
-                if (!ptrInventory.getSlot(slot).isEqual(it))
+                if (ptrInventory.getSlot(slot) != it)
```

Comment:
For now, client code fixes like this one should probably be on the 0.6.3 branch to avoid having another commit there that does the same thing.